lib/AstOfLlbc.ml

```diff
@@ -566,47 +680,190 @@ let maybe_addrof (env: env) (ty: C.ty) (e: K.expr) =
   | _ ->
       K.(with_type (TBuf (e.typ, false)) (EAddrOf e))
 
+
+(** Handling trait clauses as dictionaries *)
+
+(* There are two ways that we skip synthesis of trait methods in function calls. The first one is if
+  a trait declaration is blocklisted. This happens if the trait has built-in support (e.g.
+  FnMut), or if the trait relies on unsupported features. The second way we skip trait methods
```

Comment:
"relies on unsupported features": do you have an example of trait method that you skip this way?

Reply:
added an example of a trait method that I skip